risczero/methods/guest/Cargo.toml

```diff
@@ -5,12 +5,15 @@ edition = "2021"
 
 [workspace]
 
-[dependencies.risc0-zkp]
-version = "1.0.0-rc.2"
+[build-dependencies.risc0-build]
```

Comment:
I don't think the guest needs to depend on `risc0-build`

Reply:
Oh, great. Not sure how I came to believe it was needed.